.ts-eslintrc.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     ],
     rules: {
         "@typescript-eslint/no-floating-promises": 2,
-        "@typescript-eslint/no-misused-promises": 2
+        "@typescript-eslint/no-misused-promises": 2,
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": ["warn"],
```

Comment:
See https://github.com/import-js/eslint-plugin-import/issues/653#issuecomment-840228881 (I was getting the error in the OP of that thread when trying to lint)